README.md

```diff
@@ -6,13 +6,100 @@ via a dracut module.
 
 
 ## Kernel command line options for coreos-installer running in the initramfs
+
 * coreos.inst=yes - Instruct the installer to run
 * coreos.inst.install_dev - The block device on the system to install to
 * coreos.inst.image_url - The url of the coreos image to install to this device
 * coreos.inst.ignition_url - The url of the coreos ignition config (optional, enter
   coreos.inst.ignition_url=skip to not load an ignition config)
 
-## Testing out the installer script
+## Using the installer on Fedora or RHEL CoreOS
+
+This installer is incorporated into Fedora CoreOS and RHEL CoreOS.
+There are ISO images that can be downloaded that will allow for an
+install to be performed on bare metal hardware either via ISO install
+or via a PXE install.
+
+#### Grab an ISO image and bare metal image
+
+For Fedora CoreOS you can currently grab an ISO image from the output
+of the current development pipeline located
+[here](https://ci.centos.org/artifacts/fedora-coreos/prod/builds/latest/).
+
+The ISO image can install in either legacy boot (BIOS) mode or in UEFI
+mode. You'll have to make sure you download the appropriate related
+image for the mode you'd like to use. It is a good idea to download
+them before doing the install as the artifacts server that is being
+used currently has very slow download speeds.
+
+For example download:
+
+- [fedora-coreos-29.731.iso](https://ci.centos.org/artifacts/fedora-coreos/prod/builds/latest/fedora-coreos-29.731.iso)
+
+and one of the following two:
+
+- [fedora-coreos-29.731-metal-bios.raw.gz](https://ci.centos.org/artifacts/fedora-coreos/prod/builds/latest/fedora-coreos-29.731-metal-bios.raw.gz)
+- [fedora-coreos-29.731-metal-uefi.raw.gz](https://ci.centos.org/artifacts/fedora-coreos/prod/builds/latest/fedora-coreos-29.731-metal-uefi.raw.gz)
+
+**NOTE** Those links will quickly become broken because we prune
```

Comment:
Since we're going to merge bios/uefi, we should make a note that these are just unstable in general until we have an actual release.

Reply:
yeah I think we'll just update this when we move to official docs. I can add a note here about these being development images, though